sensor/common/sensor/sensor.go

```diff
@@ -217,11 +217,10 @@ func (s *Sensor) Start() {
 
 	select {
 	case <-errSig.Done():
-		s.stoppedSig.SignalWithErrorWrap(errSig.Err(), "getting connection from connection factory")
+		s.stopper.Flow().StopWithError(errors.Wrap(errSig.Err(), "getting connection from connection factory"))
+		s.stopper.Flow().ReportStopped()
 		return
 	case <-okSig.Done():
-	case <-s.stoppedSig.Done():
-		return
```

Comment:
This was simply redundant because nothing modified `s.stoppedSig` before this point, and `Sensor.Stop()` doesn't alter it either (maybe it should, but `Sensor.Stop()` looks like a can of worms at the moment).